py/preprocess_data.py

```diff
@@ -38,6 +41,15 @@
 # )
 # Merge to get question text.
 responses2 = responses2.merge(questions, on="question_id")
+# merge to get val_order
```

Comment:
nit: please capitalize first word of comments per PEP8 (and our [new style guide](https://ubicenter.dev/contributors/code/style.html#comments))